libcxx/CMakeLists.txt

```diff
@@ -177,6 +177,10 @@ set(LIBCXX_LIBDIR_SUFFIX "${LLVM_LIBDIR_SUFFIX}" CACHE STRING
     "Define suffix of library directory name (32/64)")
 option(LIBCXX_INSTALL_HEADERS "Install the libc++ headers." ON)
 option(LIBCXX_INSTALL_LIBRARY "Install the libc++ library." ON)
+cmake_dependent_option(LIBCXX_INSTALL_MODULE_INTERFACE_UNIT_SOURCES
```

Comment:
I'll be 100% honest, `LIBCXX_INSTALL_MODULE_INTERFACE_UNIT_SOURCES` reads like jibberish to me. I understand it's the correct term, but it is long and harder to read. I don't think `LIBCXX_INSTALL_STD_MODULES` was  confusing. This comment is non-blocking, but personally I like the simple and "dumb" `LIBCXX_INSTALL_STD_MODULES` better.

FWIW I think part of my dislike for `LIBCXX_INSTALL_MODULE_INTERFACE_UNIT_SOURCES` is that it's so long that it's difficult to distinguish from `LIBCXX_INSTALL_MODULE_INTERFACE_UNIT_SOURCE_DIR`, and it's also not clear that it's a boolean option (since there's the word `SOURCES` in there). `LIBCXX_INSTALL_STD_MODULES` makes it clear that this is a `ON`/`OFF` toggle.